webui/react/src/components/kit/utils/components/Link.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { CSSProperties, MouseEvent, useCallback } from 'react';
+
+import Button from 'components/kit/Button';
+import { AnyMouseEventHandler } from 'components/kit/utils/types';
+import { handlePath, linkPath } from 'routes/utils';
+
+import css from './Link.module.scss';
+
+const windowOpenFeatures = ['noopener', 'noreferrer'];
+
+export interface Props {
+  children?: React.ReactNode;
+  className?: string;
+  disabled?: boolean;
+  // is this external to the assets hosted by React?
+  external?: boolean;
+  inherit?: boolean;
+  isButton?: boolean;
+  label?: string;
+  onClick?: AnyMouseEventHandler;
+  path?: string;
+  popout?: boolean | 'tab' | 'window';
+  size?: 'tiny' | 'small' | 'medium' | 'large';
+  style?: CSSProperties;
+}
+
+const Link: React.FC<Props> = ({ external, popout, onClick, ...props }: Props) => {
```

Comment:
Can you make a ticket to add `Link` to the UI Kit? Seems like something we need.